src/zip.c

```diff
@@ -957,3 +957,129 @@ int zip_extract(const char *zipname, const char *dir,
 
   return status;
 }
+
+int zip_extract_stream(const char *zipstream, size_t size, const char *dir,
```

Comment:
Could you rename this arg, to just `stream`

Reply:
what do you mean? 
 Variable： void * arg ?？

Reply:
Do you mean zipstream -> stream?

Reply:
In line 602 : FILE *stream = NULL;
arg stream means FILE *. 
so I named it zipstream.

Reply:
yeah, but it's a different function and local var. so we can rename to something else.

Reply:
Ok.